MinecraftClient/McClient.cs

```diff
@@ -3154,6 +3154,22 @@ public void OnDestroyEntities(int[] Entities)
         /// <param name="Dy"></param>
         /// <param name="Dz"></param>
         /// <param name="onGround"></param>
+        public void OnEntityPosition(int EntityID, Double Dx, Double Dy, Double Dz, byte _yaw, byte _pitch, bool onGround)
```

Comment:
Please update method document comment

Reply:
I have added the required parameters in the documentation.